src/v/cluster/feature_manager.cc

```diff
@@ -302,7 +303,8 @@ ss::future<> feature_manager::do_maybe_update_active_version() {
     // This call in principle can be a network fetch, but in practice
     // we're only doing it immediately after cluster health has just
     // been updated, so do not expect it to go remote.
-    auto node_status_v = co_await _hm_frontend.local().get_nodes_status({});
```

Comment:
exactly it was the negative value so it was already in the past

Reply:
>As an aside, I think we really should reconsider the habit of leaving parameters with common types (such as timepoint) unnamed. The parameter name should be easy to peek in an IDE.

@ztlpn can you give an example of what you mean? afaict the only time we have unnamed parameters is to avoid unused variable warning from compiler (but we could start using [[maybe_used]]) if that is what you are referring to.

Reply:
I mean in the .h files:
```
    // Return status of all nodes
    ss::future<result<std::vector<node_state>>>
      get_nodes_status(model::timeout_clock::time_point);
```

What is the meaning of the parameter? Only by checking the .cc file you can find out that it is the deadline.